packages/certs-v5/lib/domains.js

```diff
@@ -66,4 +66,37 @@ function printDomains (domainsTable, msg) {
   }
 }
 
-module.exports = {waitForDomains, printDomains}
+async function waitForCertIssuedOnDomains(context, heroku) {
+  function certIssuedOrFailedForAllCustomDomains (domains) {
+    domains = domains.filter((domain) => domain.kind === 'custom')
+    return _.every(domains, (domain) => domain.acm_status === 'cert issued' || domain.acm_status === 'failed')
+  }
+
+  function someFailed (domains) {
+    domains = domains.filter((domain) => domain.kind === 'custom')
+    return _.some(domains, (domain) => domain.acm_status === 'failed')
+  }
+
+  function apiRequest (context, heroku) {
+    return heroku.get(`/apps/${context.app}/domains`)
+  }
+
+  let domains = await apiRequest(context, heroku)
+
+  if (!certIssuedOrFailedForAllCustomDomains(domains)) {
+    await cli.action('Waiting until the certificate is issued to all domains', (async function () {
+      let i = 0;
+      do {
```

Comment:
Given that `apiRequest` is called once before we even hit this loop, I believe we could simplify this a bit by converting it to a simple `while` loop:
```js
while (!certIssuedOrFailedForAllCustomDomains(domains)) {
  await wait(60*1000)
  domains = await apiRequest(context, heroku)
}
```

Reply:
Nice! 👍🏼

Reply:
Yeah, you're right, that's better